Documentation/PageTsconfig/TceMain.rst

```diff
@@ -295,14 +295,14 @@ set, the page access permissions are copied from the parent page.
 Example: Inherit the group id of the parent page
 """"""""""""""""""""""""""""""""""""""""""""""""
 
-..  code-block:: php
-    :caption: config/system/settings.php | typo3conf/system/settings.php
-
-    $GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPageTSconfig'] .= '
-        TCEMAIN.permissions.groupid = copyFromParent
-        TCEMAIN.permissions.group = 31
-        TCEMAIN.permissions.everybody = 0
-    ';
+..  code-block:: typoscript
```

Comment:
Shouldn't we backport this also to 13.4 to avoid deprecated code (maybe as separate PR)?